os/board/common/crashdump.c

```diff
@@ -167,6 +358,35 @@ static int ramdump_via_uart(void)
  ****************************************************************************/
 
 #if defined(CONFIG_BOARD_CRASHDUMP)
+void crashdump_init(void)
+{
+	int index;
+
+	index = get_dump_partindex("coredump");
+	if (index < 0) {
+		lldbg("partition index is wrong for coredump part name\n");
+		return;
+	}
+	snprintf(coredump_partname, MTD_PARTNAME_LEN + 1, "/dev/mtdblock%d", index);
+
+	coredump_partsize = get_dump_partsize(index, &coredump_partoffset);
+	if (!coredump_partsize) {
+		lldbg("partition index is wrong for coredump part name\n");
+	}
+
+	index = get_dump_partindex("ramdump");
+	if (index < 0) {
+		lldbg("partition index is wrong for ramdump part name\n");
+		return;
+	}
+	snprintf(ramdump_partname, MTD_PARTNAME_LEN + 1, "/dev/mtdblock%d", index);
```

Comment:
same as previous

Reply:
It will be used for reading through mtd layer by tools in future.